uber/models.py

```diff
@@ -1324,6 +1348,22 @@ def past_years_json(self):
         return json.loads(self.past_years or '[]')
 
 
+class WatchList(MagModel):
```

Comment:
I don't think we want ``uselist=False`` here because multiple attendees could map to the same watchlist entry, right?